hack/examples/patches/eks/initialize-variables.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 Nutanix. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+- op: "add"
+  path: "/spec/topology/variables"
+  value:
+    - name: "clusterConfig"
+      value:
+        addons:
+          clusterAutoscaler: {}
+          nfd: {}
+          cni:
+            provider: Cilium
+            values:
+              sourceRef:
+                name: ${CLUSTER_NAME}-cilium-cni-helm-values-template
+                kind: ConfigMap
```

Comment:
This is already added with `cilium-with-custom-values.yaml`

Reply:
That file `cilium-with-custom-values.yaml` is adding the actual ConfigMap, here we're configuring the addon to use that ConfigMap.